assets/js/script.js

```diff
@@ -0,0 +1,125 @@
+// Preloader js    
```

Comment:
Why do we need to import the whole file?

Reply:
I thought it was a good idea because:
- I need this to happen on all pages
- Since we are not having these floating SVGs for now, overwriting the original JS file avoids JS errors on the console.

But now I see that I could do it in a separate file to have only this behavior, but then we won't get rid of the JS console errors.

What do you think?

Reply:
I was assuming that once we get the missing graphic from the designer we want it to have this parallax effect? In this case we can just keep the errors and hope the graphic is being delivered soon 😉

Reply:
And even if we want to have the floating SVG images, I am pretty sure that would end up changing the script to make them behave differently or we are going to have less images than the ones the script expects.

Reply:
Or if I want this to happen in all pages, I would need to add the script to all pages or to add it to the `footer.html`. 
But still, after seeing the JS code and the `index.html` I really think we will end up modifying the behavior of the `parallax` part.

Reply:
feel free to move the script in as we still can get rid of it if we end up not modifying things. Just make sure to clean it a bit up, e.g. replace tabs with spaces to make the linter happy.